test/Directory.Build.props

```diff
@@ -1,15 +1,16 @@
 <Project>
 
   <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))"/>
-
+  <PropertyGroup>
+    <EnableMSTestRunner>true</EnableMSTestRunner>
+    <TestingPlatformDotNetTestSupport>true</TestingPlatformDotNetTestSupport>
+  </PropertyGroup>
   <ItemGroup Label="Package References">
     <PackageReference Include="coverlet.collector">
```

Comment:
This package doesn't work with MTP, you need to switch to Microsoft code coverage (recommended for perf) or use coverlet DotNet tool

Reply:
@Evangelink It's unused. CI looks like it runs `dotnet test` without parameters.